CourseManager/Courses/Course 1 Build a SwiftUI app for iOS 15/Views/Detail/CourseView.swift

```diff
@@ -197,9 +207,9 @@ struct CourseView: View {
     }
 
     func fadeOut() {
-        appear[0] = false
-        appear[1] = false
-        appear[2] = false
+        for i in 0 ..< appear.count {
```

Comment:
It can be written in shorter way
```
appear = appear.map { false }

appear = Array(repeating: false, count: appear.count)
```

Reply:
I've updated to appear = Array(repeating: false, count: appear.count). Thank you, spent a while trying to figure out better code for this earlier, wasn't aware of this.